challenge_6/python/ning/challenge_6.py

```diff
@@ -0,0 +1,19 @@
+def ranges(input_list):
+    output_elements = list(input_list)  # copy, not assign
+    output_list = []
+
+    for integer in input_list:
+        if (integer + 1 not in input_list and
```

Comment:
Thanks for the suggestions, TIL about sets. This portion of the code has been updated accordingly in 15d3cbd.